app/models/app_setting.rb

```diff
@@ -6,6 +6,9 @@ class AppSetting < ApplicationRecord
   validates :github_user_name, :git_repository_name, :git_repository_url, presence: true
   validates :email_notification_smtp_port, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 65_535 }, allow_nil: true
   validates :vm_archivation_timeout, numericality: { greater_than_or_equal_to: 0 }
+  validates :vsphere_root_folder,
+            length: { maximum: 79 },
```

Comment:
Is there a vSphere constraint that limits the folder name to 79 characters?

Reply:
Exactly.

Reply:
Can we define a constant VSPHERE_FOLDER_NAME_CHARACTER_LIMIT, which stores the value to avoid magic numbers?

Reply:
Done!